wazo_dird/plugins/csv_backend/plugin.py

```diff
@@ -105,6 +111,16 @@ def _load_file(self):
         except IOError:
             logger.exception('Could not load CSV file content')
 
+    def _check_for_csv_file_update(self, filename):
+        fname = pathlib.Path(filename)
+        csv_file_last_modification_date = datetime.datetime.fromtimestamp(
+            fname.stat().st_mtime
+        )
+        current_date = datetime.datetime.today()
```

Comment:
I don't think you need to do any time arithmetic. If `csv_file_last_modification_date` is different from the last modification time from the last time the file got loaded you can re-read the file and save the modification time for your next check.